gateway/src/project.rs

```diff
@@ -615,9 +615,7 @@ pub struct Service {
 
 impl Service {
     pub fn from_container(container: ContainerInspectResponse) -> Result<Self, ProjectError> {
-        let container_name = safe_unwrap!(container.name.strip_prefix("/")).to_string();
-
-        let resource_name = safe_unwrap!(container_name.strip_suffix("_run")).to_string();
+        let name = safe_unwrap!(container.config.labels.get("project.name")).to_string();
```

Comment:
@brokad I tried to trace where `Service::name` will be used but it seems to be only for the health check. However, can you still confirm I'm not breaking anything here?

Reply:
You may be breaking existing containers that were not deployed with the label.

Reply:
Thanks @brokad!! I've changed it to a backwards compatible implementation